examples/atomspace/python.scm

```diff
@@ -29,6 +29,7 @@
 (python-eval "
 from opencog.atomspace import AtomSpace, TruthValue
 from opencog.atomspace import types
+from opencog.utilities import set_type_ctor_atomspace
```

Comment:
In the unit tests at the bottom, we have `from opencog.type_constructors import atomspace` which imports the single global atomspace.   But here, we have to use `opencog.utilities` to set that single global atomspace. Shouldn't the setter and the getter live in the same place?